envs/example/defaults-2.0.yml

```diff
@@ -584,8 +584,8 @@ common:
     mcelog: False
   ursula_monitoring:
     method: tar # git|tar
-    tar_url: https://file-mirror.openstack.blueboxgrid.com/ursula-monitoring/2.2.5.tar.gz
-    tar_version: 2.2.5
+    tar_url: https://file-mirror.openstack.blueboxgrid.com/ursula-monitoring/2.2.11.tar.gz
+    tar_version: 2.2.11
```

Comment:
@paulczar Did you want me to switch this to pull in the master branch of ursula-monitoring?